doc/install/index.rst

```diff
@@ -1,42 +1,52 @@
+.. include:: ../links.inc
+
 Quick start
 ===========
 
-MNE-Python requires Python version 3.7 or higher.
+MNE-Python requires Python version |min_python_version| or higher. If you've
+never worked with Python before, skip ahead to the last paragraph of this page.
 For users already familiar with Python:
 
 - If you only need 2D plotting capabilities with MNE-Python (i.e., most EEG/ERP
   or other sensor-level analyses), you can install MNE-Python using ``pip``:
 
   .. code-block:: console
 
-      $ pip install mne  # dependencies are numpy, scipy, matplotlib
+      $ pip install mne
+
+  The only hard dependencies are `NumPy`_ and `SciPy`_, though most users will
+  want to install `Matplotlib`_ too (for plotting).
 
-- If you need MNE-Python's 3D plotting capabilities (e.g., plotting estimated
+- If you need MNE-Python's 3D rendering capabilities (e.g., plotting estimated
   source activity on a cortical surface) it is a good idea to install
-  MNE-Python into its own virtual environment. To do this with ``conda`` (this
-  will create a conda environment called ``mne``):
+  MNE-Python into its own virtual environment. To do this with
+  `conda <anaconda>`_:
 
   .. code-block:: console
 
-      $ curl --remote-name https://raw.githubusercontent.com/mne-tools/mne-python/master/environment.yml
-      $ conda env update --file environment.yml
+      $ conda create --name mne --channel conda-forge mne
+      $ #                   ↑↑↑                       ↑↑↑
+      $ #             environment name            package name
 
+  This will create a new ``conda`` environment called ``mne``.
   If you need to convert structural MRI scans into models of the scalp,
   inner/outer skull, and cortical surfaces you also need
   :doc:`FreeSurfer <freesurfer>`.
 
-The :ref:`install_python_and_mne_python` page has more detailed instructions
-for different operating systems (including instructions for installing Python
-if you don't already have it). The :ref:`advanced_setup` page has additional
+The :ref:`standard_instructions` page has detailed instructions for different
```

Comment:
For continuity with the first paragraph, and to make the next link seem more new user friendly

```suggestion
For users unfamiliar with python, the :ref:`standard_instructions` page has detailed instructions for different
```

Reply:
Python with upper case P

Reply:
But why this change? This is not about Python, it's about Freesurfer.

Reply:
The page I edited is entirely about is about python and nothing to do with FreeSurfer. And the link is to how to a detailed instructions on how install python. Am I confused?

Reply:
Maybe I was confused. You're right, of course this is referring to Python installation instructions, so your suggestion is fine (but at least my comment on the capital P is still correct 😝 ).

Reply:
That `P` gets me every time!